examples/MySQL/dialect.yml

```diff
@@ -122,21 +203,102 @@ sql functions:
       CASE WHEN $1 > 12 OR $1 < 1 OR $2 > 31 OR $2 < 1 THEN NULL ELSE
       Date(STR_TO_DATE(CONCAT($0,'-',$1,'-',$2),
               CASE WHEN length($0) > 2 THEN '%Y-%m-%d' ELSE '%y-%m-%d' END)) END
+# Description: Creates a date from string 2 possible string date format masks  'YYYY-MM-DD' and 'YY-MM-DD'   
```

Comment:
Should this be `Creates a date from strings. Two possible string date format masks:  'YYYY-MM-DD' and 'YY-MM-DD'`   ?